src/main/perl/CLI.pm

```diff
@@ -313,6 +313,9 @@ sub _initialize
     $self->init_history($self->option("history-instances"))
         if $self->option("history");
 
+    # Setup the verbose_logfile option
+    $self->setup_reporter(undef, undef, undef, undef, 1);
```

Comment:
Since this could be very noisy I think we should make this configurable. It's useful/recommended but if people have nodes with small partitions for logging then it could break things for them, so I would suggest it be configurable and off by default.